packages/admin/src/pages/build-application/[applicationId]/[sectionId]/edit-title.page.tsx

```diff
@@ -16,12 +16,15 @@ export const getServerSideProps = (context: GetServerSidePropsContext) => {
     const { sectionTitle } = application.sections.find(
       (section) => section.sectionId === sectionId
     )!;
+    const {
+      audit: { version },
+    } = application;
 
-    return { sectionTitle, applicationId, sectionId };
+    return { sectionTitle, applicationId, sectionId, version };
   };
 
   const handleRequest = async (
-    body: { sectionTitle: string },
+    body: { sectionTitle: string; version: string },
     sessionId: string
```

Comment:
Not that fussed but instead of having a hidden input it might be cleaner to just grab it from the pageData using the prop: `pageData` in the `handleRequest` function